tutorials/katacoda/2-serving-intro-yaml/step3.md

```diff
@@ -0,0 +1,68 @@
+## Blue/Green deploy
+The Knative `service` resource creates additional resources "route, configuration and revision" to manage the lifecycle of the application.
```

Comment:
I have similar feedback I gave back in [the cli section](https://github.com/knative/docs/pull/3142/files#r599149635)

Reply:
fixed!